### DISCUSSION TITLE
Aaron Graybill - The Simulation With Balls

### DISCUSSION THREAD

**Discussion by giscus** [aarongraybill/personal_website#6] [General]
# Aaron Graybill - The Simulation With Balls

Simulating the scoring system of The Quiz With Balls to understand if it is fair.

https://www.aarongraybill.com/mini_projects/quiz_with_balls/quiz_with_balls.html

**Comment by gsekeres** [comment]
This is incredibly interesting, and it's really cool that the empirics broadly match your predictions. One thing I'd want to know -- my intuition is that each question has some number of "easy" correct answers and some number of "hard" correct answers. One attempt to balance the advantage of having an extra player early could be by eliminating the "hard" answers first, leaving only the easy answers. If this is doable, would you be able to simulate team $A$ having 5/6 answers for 5 players, and team $B$ having 4/5 answers for 4 players? This would (ham-fistedly) represent $B$ having access to an "easy" answer. My intuition is that it won't change, but it would be interesting to see.

**Reply by aarongraybill** [reply]
That's an interesting point. There is definitely some variance in the difficulty between the possible options. So it may be that when you're losing players, you now only have to allocate players to the more confident slots. What you're suggesting could work (and would definitely be the easiest way to slot it into the above model). But given the limitations on how tractable the model is, I think you'd be better off mostly abandoning the mathematical derivations and giving a making the simulation more robust (allowing players to have different skill levels, different difficulty of questions, etc).
